Geometry/HcalTowerAlgo/src/HcalGeometry.cc

```diff
@@ -11,7 +11,7 @@ typedef CaloCellGeometry::Pt3D     Pt3D     ;
 typedef CaloCellGeometry::Pt3DVec  Pt3DVec  ;
 typedef CaloCellGeometry::Tr3D     Tr3D     ;
 
-//#define EDM_ML_DEBUG
+#define EDM_ML_DEBUG
```

Comment:
is this removed already in a follow up PR?

Reply:
I see that it was actually fixed in 
4297d2489bfac27c8fc19d37d58b5264f424ecfe
was there a PR?